clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -54,7 +54,7 @@ const ConsentBanner: FunctionComponent<BannerProps> = ({
     if (bannerIsOpen) {
       onOpen();
     }
-  }, [bannerIsOpen]);
+  }, [bannerIsOpen, onOpen]);
```

Comment:
also noticed a few places where the dependency array wasn't complete

Reply:
this _does_ mean that the `onOpen` function needs to be wrapped in a `useCallback` in order to avoid infinite renders!